### DISCUSSION TITLE
How to upgrade to latest version?

### DISCUSSION THREAD

**Discussion by clairem-sl** [pal1000/mesa-dist-win#167] [Q&A]
First of all, I use the "per-app deployment" mode rather than "whole-system" mode.

Since "per-app deployment" creates symlinks, I guess I can upgrade by simply extracting the newest version over the older version, and overwriting existing files?

Or should I first remove all per-app deployments, replace the files, and redo the deployment?

**Comment by pal1000** [comment]
> Since "per-app deployment" creates symlinks, I guess I can upgrade by simply extracting the newest version over the older version, and overwriting existing files?

Yes or delete old version and extract new version in same location.  Leave symlinks untouched.
